src/illumidesk/illumidesk/authenticators/authenticator.py

```diff
@@ -163,6 +169,48 @@ async def setup_course_hook(
     return authentication
 
 
+async def setup_user_hook_auth0(
```

Comment:
Would the custom authenticator also need it's own Class that inherits from Oauthenticator in order to decode jwt claims?

Reply:
I think it should not be needed, this method is more list a post auth hook